wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -236,51 +269,63 @@ function register_settings() {
 }
 add_action( 'admin_init', 'register_settings' );
 
-/* Callback for a custom setting */
+/**
+ *  Callback for a custom setting
+ *  */
 function show_checkbox_country() {
     $option  = get_option( 'show_settings' );
     $checked = '';
     if( $option["show_country"] == 1 ) $checked = 'checked';
     echo '<input type="checkbox" name="country_box" value="1"  ' . $checked . '  />';
```

Comment:
Another inline IF and definitely see how `checked()` works: https://developer.wordpress.org/reference/functions/checked/